platform/osx/display_server_osx.h

```diff
@@ -228,7 +228,7 @@ class DisplayServerOSX : public DisplayServer {
 	virtual float screen_get_scale(int p_screen = SCREEN_OF_MAIN_WINDOW) const override;
 	virtual float screen_get_max_scale() const override;
 	virtual Rect2i screen_get_usable_rect(int p_screen = SCREEN_OF_MAIN_WINDOW) const override;
-
+	virtual float screen_get_refresh_rate(int p_screen = SCREEN_OF_MAIN_WINDOW) const override;
```

Comment:
Same here, please don't remove separation lines between blocks of semantically unrelated methods.

Reply:
Gotcha, I'll think more about stuff like that in future PRs. Thanks!